src/models.ts

```diff
@@ -55,8 +56,8 @@ export enum SortField {
   'alltimeview' = 'alltimeview',
   'weeklyview' = 'weeklyview',
   'title' = 'title',
-  'datearchived' = 'datearchived',
   'date' = 'date',
+  'datearchived' = 'datearchived',
   'datereviewed' = 'datereviewed',
   'dateadded' = 'dateadded',
   'creator' = 'creator',
```

Comment:
The order of these should be consistent, as it is how they appear in the mobile-view dropdown.